src/priceClient/adapters/acrossApi.ts

```diff
@@ -39,6 +39,8 @@ export class PriceFeed extends BaseHTTPAdapter implements PriceFeedAdapter {
 
   private validateResponse(response: unknown): response is AcrossPrice {
     if (typeof response !== "object") return false;
-    return response !== null && typeof (response as { [key: string]: unknown }).price === "number";
+    const responseMapped = response as { price: number } | null;
+    const priceMapped = Number(responseMapped?.price);
+    return response !== null && !isNaN(priceMapped) && priceMapped > 0;
```

Comment:
One way of resolving this could be to add a new "default adapter" that just returns 0 for each mapping. This would permit the caller to easily control whether they want to throw on price resolution failure, or to default to something else. This would also provide an interesting opportunity for setting non-zero default prices for specific tokens, which is something we've actually needed in the API and was one reason that the API was unable to migrate to using the price client.

PR here: https://github.com/across-protocol/sdk/pull/774

Reply:
Resolving this now that #774 has been tested and merged :+1: